install-online/profile.hbs.md

```diff
@@ -297,8 +297,7 @@ buildservice:
     namespace: "KP-DEFAULT-REPO-SECRET-NAMESPACE"
 
 tap_gui:
-  service_type: ClusterIP # If the shared.ingress_domain is set as earlier, this must be set to ClusterIP.
-  metadataStoreAutoconfiguration: true # Create a service account, the Kubernetes control plane token and the requisite app_config block to enable communications between Tanzu Application Platform GUI and SCST - Store.
+  metadataStoreAutoconfiguration: true # Creates a service account, the Kubernetes control plane token and the requisite app_config block to enable communications between Tanzu Application Platform GUI and SCST - Store.
```

Comment:
SCST - supply chain security tools (took me back to the TAP Docs to decipher that one)

Reply:
I think it's the standard nomenclature for metadata store (which also has the name AMR apparently)

```shell
$ ag 'SCST - Store' | wc -l
117
```

Reply:
Anyways the only reason that line is appearing in the diff at all is because I fixed a typo that fell through the cracks.

It used to be "Create a service account" and I made it "Creates a service account"